packages/dd-trace/test/config.spec.js

```diff
@@ -277,7 +275,7 @@ describe('Config', () => {
     expect(config).to.have.nested.property('experimental.exporter', 'log')
     expect(config).to.have.nested.property('experimental.enableGetRumData', true)
     expect(config).to.have.nested.property('appsec.enabled', true)
-    expect(config).to.have.nested.deep.property('appsec.rules', RULES_JSON)
+    expect(config).to.have.nested.deep.property('appsec.rules', RULES_JSON_PATH)
```

Comment:
`RULES_JSON_PATH` is a string right ? so no need for `.deep`, that's for object comparison